src/ui/features/projects/ProjectForm/steps/DescriptionStep/DescriptionStep.tsx

```diff
@@ -83,6 +84,7 @@ const TextField: React.FC<TextFieldProps> = (props) => {
           lineHeight="xs"
           view="alert"
           className={cnDescriptionStep('ErrorText').toString()}
+          data-testid={errorTestId}
```

Comment:
```suggestion
          data-testid={`${props['data-testid']}:error`}
```
И не нужно будет добавлять ещё один пропс и поддерживать его.

Reply:
в таком случае придётся добавить проерку пропса иначе получим `:error` в любом случае, так что лучше проконтролировать пропс

Reply:
Это да, а то получим `undefined:error`. Мое предложение было к тому, чтобы не плодить `testIDSomething`, а передать только один пропс и внутри сделать проверку и добавление остальных `testid`.